README.md

```diff
@@ -473,6 +473,15 @@ If SSL Termination is not available, the PEP Proxy can be configured to listen H
 * In the `config.js` file, change the `config.ssl.active` flag to true.
 * In the same ssl object in the configuration, fill the path to the key and cert files.
 
+### Multi-instance configuration
+PEP Proxy is able to start multiple instances by adding and configuring certain files in `/etc/pepProxy` and using `pepProxy` service script
```

Comment:
Just a minor detail, but "active folders" like this one usually end with a ".d" suffix in unix systems. Maybe you should add that.